aries_cloudagent/multitenant/base.py

```diff
@@ -48,7 +48,10 @@ def __init__(self, profile: Profile):
         if not profile:
             raise MultitenantManagerError("Missing profile")
 
-        self._instances: dict[str, Profile] = {}
+    @abstractproperty
+    def open_profiles(self) -> Iterable[Profile]:
+        """Return iterator over open profiles."""
+        ...
```

Comment:
Ellipsis?

Reply:
Apparently it's about the same as writing 'pass', I just haven't seen it much before.